RLBotFramework/utils/item_id_parser.py

```diff
@@ -0,0 +1,120 @@
+# data in the form of: ID,FullName,AssetPackageName,AsciiLabel,PackID,Quality,UnlockMethod
+
+dump_file=input('Data dump (input) file name?->')
+data = open(dump_file, 'r', encoding='utf-8-sig')
+Bodies,Wheels,Boosts,Antennas,Decals,Toppers,Trails,Goal_Explosions,Paints,Banners,Engine_Audio=[],[],[],[],[],[],[],[],[],[],[]
+
+car_dict={'backfire': 'Backfire', 'force': 'Breakout', 'octane': 'Octane', 'orion': 'Paladin', 'rhino': 'Road Hog', 'spark': 'Gizmo', 'torch': 'X-Devil', 'torment': 'Hotshot', 'vanquish': 'Merc', 'venom': 'Venom', 'import': 'Takumi', 'musclecar': 'Dominus', 'zippy': 'Zippy', 'scarab': 'Scarab', 'interceptor': 'Ripper', 'wastelandtruck': 'Grog', 'musclecar2': 'Dominus GT', 'torch2': 'X-Devil Mk2', 'neocar': 'Masamune', 'rhino2': 'Road Hog XL', 'takumi': 'Takumi RX-T', 'aftershock': 'Aftershock', 'neobike': 'Esper', 'marauder': 'Marauder', 'carcar': 'Breakout Type-S', 'number6': 'Proteus', 'cannonboy': 'Triton', 'o2': 'Octane ZSR', 'gilliam': 'Vulcan', 'bone': 'Bone Shaker', 'scallop': 'Twin Mill III', 'endo': 'Endo', 'charged': 'Ice Charger', 'flatbread': 'Mantis', 'pumpernickel': 'Centio V17', 'focaccia': 'Animus GP', 'challah': "'70 Dodge Charger R/T", 'melonpan': "'99 Nissan Skyline GT-R R34", 'levain': 'Werewolf', 'greycar': 'DeLorean Time Machine', 'interceptor': 'Interceptor_SF', 'darkcar': "'16 Batmobile", 'takumi_ii': 'Takumi RX-T', 'sourdough': 'Jäger 619 RS', 'multigrain': 'Imperator DT5', 'berry': 'The Dark Knight Rises Tumbler', 'eggplant': "'89 Batmobile", 'Universal Decal': 'Universal Decal'}
```

Comment:
can you make this multiple lines so we do not have to scroll very far to see it all